### PR TITLE
Remove `nvme` dependency

### PR DESCRIPTION
**Pull request by metaspace**
Running `nvme` requires root. We can get the info we need in ways that do not require root (if device is owned by user).

### CODE REVIEW

File: benchs/base.py [hunk 1]
```diff
@@ -105,6 +105,14 @@ def get_zone_capacity_mb(self, dev):
 
         return int(zonecap * bs / 1024 / 1024)
 
+    def get_drive_size_gb(self, path):
+        filename = path + "/nvme_id-ns.txt"
```
Comment (MaisenbacherD) [inline benchs/base.py]: That file should not exist anymore. See 3999b4b3967ab5d438d00cd945db6110d662e89f

File: benchs/fio_zone_writes.py [hunk 2]
```diff
@@ -52,9 +52,9 @@ def teardown(self, dev, container):
 
     def report(self, path):
 
-        devsize = self.get_drive_size_gb(path)
+        devsize = self.get_nvme_drive_capacity_gb(path)
```
Comment (MaisenbacherD) [inline benchs/fio_zone_writes.py:55]: Should `devsize` also be renamed in `devcap`?

File: benchs/rocksdb.py [hunk 3]
```diff
@@ -91,9 +91,9 @@ def get_result_from_test(self, filename, testname):
                 return [i for i in line.split(' ') if i]
 
     def report(self, path):
-        devsize = self.get_drive_size_gb(path)
+        devsize = self.get_nvme_drive_capacity_gb(path)
```
Comment (MaisenbacherD) [inline benchs/rocksdb.py:94]: Same here: Should `devsize` also be renamed in `devcap`?

**Review by MaisenbacherD — COMMENTED** [review]
Made a couple of comments. Otherwise it looks good to me :)

File: benchs/base.py (hunk 1 above)
Comment (metaspace) [reply benchs/base.py]: It is renamed by the next patch in the series.

File: benchs/fio_zone_writes.py (hunk 2 above)
Comment (metaspace) [reply benchs/fio_zone_writes.py]: fixed

File: benchs/rocksdb.py (hunk 3 above)
Comment (metaspace) [reply benchs/rocksdb.py]: fixed

File: benchs/base.py (hunk 1 above)
Comment (MaisenbacherD) [reply benchs/base.py]: Sorry I was reading the commits in the wrong order :D

**Pull request merged**